libraries/AP_HAL_ChibiOS/hwdef/MatekH743/hwdef-bl.dat

```diff
@@ -38,6 +38,9 @@ PA12 OTG_FS_DP OTG1
 PA13 JTMS-SWDIO SWD
 PA14 JTCK-SWCLK SWD
 
+# make sure Vsx is on during bootloader
+PD10  PINIO1 OUTPUT LOW
```

Comment:
You should do the camera as well I think?

Reply:
Vsw by the way, and would be better if it was labelled something like VTX_SWITCH or something more descriptive

Reply:
done